cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapStatusEvaluator.java

```diff
@@ -0,0 +1,53 @@
+package org.opencds.cqf.fhir.cr.measure.r4;
+
+import static org.opencds.cqf.fhir.cr.measure.constant.MeasureReportConstants.MEASUREREPORT_IMPROVEMENT_NOTATION_SYSTEM;
+import static org.opencds.cqf.fhir.cr.measure.constant.MeasureReportConstants.MEASUREREPORT_MEASURE_POPULATION_SYSTEM;
+
+import org.apache.commons.lang3.tuple.MutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+import org.hl7.fhir.r4.model.Measure;
+import org.hl7.fhir.r4.model.MeasureReport;
+import org.opencds.cqf.fhir.cr.measure.enumeration.CareGapsStatusCode;
+
+/*
+Care Gaps Status Evaluator houses the algorithm logic for which Care-Gap status is applicable to a Measure Report.
+ */
+public class R4CareGapStatusEvaluator {
+
+    public CareGapsStatusCode getGapStatus(Measure measure, MeasureReport measureReport) {
```

Comment:
Please javadoc this method with a brief description of the algorithm used, especially with regard to isPositive and inNumerator